apps/api/src/unit/unit.service.ts

```diff
@@ -0,0 +1,17 @@
+import { Injectable } from '@nestjs/common';
+import { InjectRepository } from '@nestjs/typeorm';
+import { Unit } from './entity/unit.entity';
+import { Repository } from 'typeorm';
+
+@Injectable()
+export class UnitService {
+  constructor(
+    @InjectRepository(Unit)
+    private unitsRepository: Repository<Unit>,
+  ) {}
+
+  async getAllUnits(): Promise<Unit[]> {
+    const found = await this.unitsRepository.find();
```

Comment:
return this.unitsRepository.find();

Reply:
Woops, right, can simplify, thank you! 
Changed.